style.css

```diff
@@ -6,468 +6,975 @@
+    margin-bottom: 10px;
+  }
 
-.btn-container {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  text-align: center;
-  margin: 20px 0;
-}
+  .page-content {
+    display: flex;
+    flex-direction: column;
+    max-width: 100%;
+    min-height: 100%;
+    justify-items: center;
+    align-items: center;
+    justify-content: center;
+    background-image: url('./images/circle\ 1\ header@2x.jpg');
+    background-position: relative;
+    background-size: 22%;
+    background-repeat: no-repeat;
+    background-position-x: left;
+    background-position-y: top;
+  }
 
-.end-line {
-  display: block;
-  align-self: center;
-  border-bottom: 2px solid rgb(70, 67, 67);
-  margin: 10px 0 0 0;
-  width: 100%;
-}
+  .descript {
+    max-width: 507px;
+    display: flex;
+    flex-grow: 1;
+  }
 
-.email-me {
-  color: #ebebff;
-  text-decoration: none;
-  font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 20px;
-  line-height: 28px;
-  margin: 10px 0 20px 0;
-}
+  h1 {
+    font-size: 85px;
+  }
+
+  .name {
+    font-weight: 500;
+    color: #6495ed;
+    text-align: center;
+    margin: 40px 0;
+  }
+
+  .occupation {
+    color: #afabab;
+    text-align: center;
+    font-weight: 400;
+    font-size: 44px;
+    margin-bottom: 30px;
+    padding-bottom: 19px;
+  }
+
+  .descript .prof {
+    display: flex;
+    max-width: 100%;
+    flex-direction: column;
+  }
+
+  .prof p {
+    color: #b3bac5;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 19px;
+    margin: 20px;
+    line-height: 28px;
+    text-align: center;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  }
+
+  .lang-skills {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    align-items: center;
+    padding: 20px;
+    gap: 30px;
+  }
 
-.mail-line {
-  display: block;
-  border-top: 3px solid rgb(0, 0, 0);
-  margin: 0 0 7px 0;
-  width: 174px;
+  .lang-skills .lang {
+    flex-grow: 0;
+    display: flex;
+    flex-direction: column;
+  }
+
+  h5 {
+    font-size: 18px;
+    font-weight: 200;
+    color: cornflowerblue;
+    padding-left: 2px;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  }
+
+  .lang ul,
+  li {
+    color: #b3bac5;
+    font-size: 15px;
+    padding: 0 2px;
+    font-weight: 400;
+    line-height: 27px;
+    list-style-type: none;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+  }
+
+  .end-line {
+    display: block;
+    align-self: center;
+    border-bottom: 2px solid rgb(70, 67, 67);
+    margin: 10px 0 0 0;
+    width: 100%;
+  }
+
+  .back-foot {
+    background-image: url('./images/circle\ 2\ hder@2x.png');
+    background-position: relative;
+    background-size: 17%;
+    background-repeat: no-repeat;
+    background-position-x: right;
+    background-position-y: bottom;
+  }
+
+  .footer {
```

Comment:
- [ ] Kindly add "work ⬇️ " element to the footer of the top section to make it look similar to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A976).
### Figma Design           
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/178801464-d7183c52-f620-49dc-89b5-e03c031f86f4.png) 

 ### Your Design

  :-------------------------:|:-------------------------:
 ![image](https://user-images.githubusercontent.com/60618877/178801648-244e02e2-e2de-42a6-aec7-d015fef1109b.png)